.github/workflows/release.yaml

```diff
@@ -0,0 +1,42 @@
+name: Create release draft
+
+on:
+  pull_request:
+    # types: [closed]
+    branches:
```

Comment:
@hjpotter92 should we trigger this on tag only?

Reply:
yeah. i'll add further conditions, once i'm able to generate the bundled assets in the workflow.